SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -539,10 +541,19 @@ void ActorSystem::HandleActorAuthority(const Worker_EntityId EntityId, const Wor
 						Actor->RemoteRole = ROLE_AutonomousProxy;
 
 						// Flush PC interest on handover
-						if (GetDefault<USpatialGDKSettings>()->bUseClientEntityInterestQueries
-							&& GetDefault<USpatialGDKSettings>()->bRefreshClientInterestOnHandover)
+						if (GetDefault<USpatialGDKSettings>()->bUseClientEntityInterestQueries)
 						{
-							Channel->MarkInterestDirty();
+							Worker_EntityId ControllerEntityId =
```

Comment:
This seems constable
```suggestion
							const Worker_EntityId ControllerEntityId =
```